src/d3a/models/strategy/area_agents/two_sided_pay_as_clear_engine.py

```diff
@@ -107,7 +107,7 @@ def _match_offers_bids(self):
             return
         clearing_rate, clearing_energy = clearing
         if clearing_energy > 0:
-            self.owner.log.info(f"Market Clearing Rate: {clearing_rate} "
+            self.owner.log.warn(f"Market Clearing Rate: {clearing_rate} "
```

Comment:
I do not know how important it is in order to add it on the warning log (to my eyes not that important, since you can see the clearing rate from the trade rate, they should both be the same). Info looks more appropriate to me. In addition to that, if you decide to stick to warning log level, you should use the warning log function instead of the deprecated warn function.

Reply:
If you both vouch to have it in info logs, then i will change it. But clearing are not too frequent to clutter the warn_logs and it might be easier while matching trade_rate & trade_energy with clearing_rate & clearing_energy.